WinPort/src/Backend/WX/wxMain.cpp

```diff
@@ -182,6 +182,16 @@ WinState::WinState()
 	pos.x = atoi(str.c_str());
 	getline(is, str);
 	pos.y = atoi(str.c_str());
+
+	std::ifstream isconsize;
+	isconsize.open(InMyConfig("consolesize").c_str());
```

Comment:
consolesize is an old file where size was saved/restored. Using it here like this will lock all user who had old versions of far2l to same startup window size. So plz use different name, like "winsize" or "fixedsize" so it wil not break those who dont know about such file at all.

Reply:
as the position and size in winstate are saved when user does F9->Options->Save_Setup, how about using the consolesize values when x/y sizes in winstate are negative?
this will also solve the 'issue' that existing consolesize for 'old' user will still work as it was before removal if one manually sets values in winstate (can also be documented)

Reply:
I'd suggest in such case to not use consolesize and keep everything in single winstate file, so if size in winstate is negative - then treat it as characters but not pixels and don't change it on window size change.